tools/__tasks__/compile/hash/index.js

```diff
@@ -106,7 +106,7 @@ module.exports = {
                             )
                         );
                     })
-                    .then(normalisedAssetMap => // save the asset map
+                    .then((normalisedAssetMap) => // save the asset map
```

Comment:
It's the comment that's causing prettier to want to parenthesise this parameter. I'll get a minimal repro working then raise an issue. I guess this shouldn't block this PR.